zproject/settings.py

```diff
@@ -329,6 +329,22 @@ def get_secret(key):
     else:
         DATABASES['default']['OPTIONS']['sslmode'] = 'verify-full'
 
+if 'postgres' in DATABASES['default']['ENGINE']:
+    # Set True if you want to support full text search against all languages
+    # USING_PGROONGA = True
+    USING_PGROONGA = False
+else:
+    USING_PGROONGA = False
```

Comment:
This user-configurable setting for `USING_PGROONGA` should be added to zproject/local_settings_template.py (installed as /etc/zulip/settings.py) so users don't need to patch Zulip to set this setting.  You should add its default value of False to DEFAULT_SETTINGS in this file.